docs/upgrading/upgrading-from-30.md

```diff
@@ -166,6 +166,22 @@ much the same as it was in earlier versions, the syntax to define these fields d
 There is some documentation already written about [how to transform Repeater and Block fields](https://docs.bolt.cm/4.0/howto/transform-repeaters-to-sets-and-collections#jumpbutton)
 that will help you out in migrating these fields for your site.
 
+### Video fields
```

Comment:
Could you make the "fields" word singular?

Reply:
Nice catch - of course I can. :smile:

Reply:
Great!! Thank you! 👍